### PR TITLE
[zh] add contributing/_index.md

### PR DESCRIPTION
**Pull request by freezer712**
[zh] add contributing/_index.md

### CODE REVIEW

File: content/zh/docs/contributing/_index.md [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+
+## 给其他仓库做贡献
+
+OpenTelemetry 是一个开源项目，我们很乐意接受新的贡献和贡献者。有关入门信息，请参阅每个 SIG 仓库中的 CONTRIBUTING.md 文件。
+
+各个 SIG 可能还维护了除此网站之外的其他文档，但我们努力从我们的主网站提供有关使用项目的准确的总体指导。
+
+如果您看到想要改进的内容，请使用 GitHub 搜索 OpenTelemetry 组织中的所有仓库。这可以帮助您确定在哪里提交问题或 PR。每个存储库都有自己的流程和程序。在提交问题或提交 PR 之前，请阅读该仓库的 `README.md`、`CONTRIBUTING.md` 和 `code-of-conduct.md`（如果存在的话）。
+
+大多数存储库使用issue和 PR 模板。查看一些未解决的问题和 PR可以帮助了解该团队的流程。提交问题或 PR 时，请确保尽可能详细地填写模板。
```
Comment (reyang) [inline content/zh/docs/contributing/_index.md:539]:
```suggestion
大多数存储库使用 issue 和 PR 模板。查看一些未解决的问题和 PR 可以帮助了解该团队的流程。提交问题或 PR 时，请确保尽可能详细地填写模板。
```

File: content/zh/docs/contributing/_index.md [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+
+下面的指引描述了如何为OpenTelemetry文档做贡献。关于如何为 OpenTelemetry项目作出贡献，请参阅[OpenTelemetry贡献者指南](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)，其中提供了有关贡献者许可协议和行为准则的详细信息。从某种程度上讲，每种语言实现、收集器和约定 [仓库](https://github.com/open-telemetry/) 都有自己特定的贡献指南。
+
+对于文档中的问题，你可以提一个关于OpenTelemetry的问题（issue），或者向[`opentelemetry.io` GitHub 仓库](https://github.com/open-telemetry/opentelemetry.io)发起一个带有变更的拉取请求（pull request）。
+
+OpenTelemetry文档贡献者可以：
+
+- 改进现有内容。
+- 添加新的内容。
+- 更新OpenTelemetry 注册表（Registry）。
```
Comment (reyang) [inline content/zh/docs/contributing/_index.md:16]:
```suggestion
- 更新 OpenTelemetry 注册表（Registry）。
```

**Review by reyang — APPROVED** [review]

File: content/zh/docs/contributing/_index.md [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+_图 2. 使用Github发起拉取请求的流程。_
+
+1. 在发现问题的页面上，选择右侧导航面板中的**Edit this page**选项。
+2. 如果您不是该项目的成员，GitHub 会建议您创建存储库的分支。选择 **Fork this repository**。
+3. 在 GitHub 编辑器中进行更改。
+4. 填写**Propose file change**表格。
+5. 选择**Propose changes**。
+6. 选择**Create pull request**。
+7. 进入**Open a pull request**页面，填写描述可以帮助评审人理解你的更改内容。
+8. 选择**Create pull request**。
```
Comment (windsonsea) [inline content/zh/docs/contributing/_index.md:113]:
```suggestion
4. 填写 **Propose file change** 表格。
5. 选择 **Propose changes**。
6. 选择 **Create pull request**。
7. 进入 **Open a pull request** 页面，填写描述可以帮助评审人理解你的更改内容。
8. 选择 **Create pull request**。
```

File: content/zh/docs/contributing/_index.md [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,1,2,3,4,5,6,7 grey
+class 0 spacewhite
+class tasks,tasks2 white
+class id1 k8s
+```
+
+_图 2. 使用Github发起拉取请求的流程。_
+
+1. 在发现问题的页面上，选择右侧导航面板中的**Edit this page**选项。
```
Comment (windsonsea) [inline content/zh/docs/contributing/_index.md:106]:
```suggestion
1. 在发现问题的页面上，选择右侧导航面板中的 **Edit this page** 选项。
```

File: content/zh/docs/contributing/_index.md [hunk 5]
```diff
@@ -0,0 +1,552 @@
...
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,1,2,3,4,5,6,7 grey
+class 0 spacewhite
+class tasks,tasks2 white
+class id1 k8s
+```
+
+_图 2. 使用Github发起拉取请求的流程。_
```
Comment (windsonsea) [inline content/zh/docs/contributing/_index.md:104]:
```suggestion
_图 2. 使用 Github 发起拉取请求的流程。_
```

**Review by windsonsea — COMMENTED** [review]
Could you provide a space between en and zh chars, to follow the style on the current pages.
You need to check from top to bottom, thanks.

File: content/zh/docs/contributing/_index.md [hunk 6]
```diff
@@ -0,0 +1,552 @@
+---
+title: 贡献
+description: 了解如何为OpenTelemetry文档做贡献。
```
Comment (windsonsea) [inline content/zh/docs/contributing/_index.md:3]:
```suggestion
description: 了解如何为 OpenTelemetry 文档做贡献。
```

File: content/zh/docs/contributing/_index.md [hunk 7]
```diff
@@ -0,0 +1,552 @@
...
+weight: 980
+default_lang_commit: 8603bc8
+---
+
+下面的指引描述了如何为OpenTelemetry文档做贡献。关于如何为 OpenTelemetry项目作出贡献，请参阅[OpenTelemetry贡献者指南](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)，其中提供了有关贡献者许可协议和行为准则的详细信息。从某种程度上讲，每种语言实现、收集器和约定 [仓库](https://github.com/open-telemetry/) 都有自己特定的贡献指南。
```
Comment (windsonsea) [inline content/zh/docs/contributing/_index.md:8]:
```suggestion
下面的指引描述了如何为 OpenTelemetry 文档做贡献。关于如何为 OpenTelemetry 项目作出贡献，请参阅
[OpenTelemetry 贡献者指南](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)，
其中提供了有关贡献者许可协议和行为准则的详细信息。从某种程度上讲，
每种语言实现、收集器和约定[仓库](https://github.com/open-telemetry/)都有自己特定的贡献指南。
```

File: content/zh/docs/contributing/_index.md (hunk 3 above)
Comment (freezer712) [reply content/zh/docs/contributing/_index.md:113]: Thanks for your reminder, I will recheck the whole document.

**Comment by reyang** [comment]
> Could you provide a space between en and zh chars, to follow the style on the current pages. You need to check from top to bottom, thanks.

Maybe a good thing to cover in the CI job.

**Review by windsonsea — APPROVED** [review]

**Review by chalin — APPROVED** [review]
The Contributing section has been reworked, but this is consistent with the given commit hash. LGTM, thanks!

**Pull request merged**